Install rust-src when it is not found (#3206)
rust-analyzer/rust-analyzer: docs/user/readme.adoc

We are not there yet, so some editor specific setup is required.

Additionally, rust-analyzer needs sources of the standard library.

> When fails to locate them, rust-analyzer attempts to install them automatically.

Reviewer: ```suggestion
If the source code is not present, rust-analyzer will attempt to it automatically.
```

Author: > will attempt to it automatically

??

Reviewer: Fixed.

Author: Thanks :)
